### ISSUE TITLE
TLDR Section only covers what COVID is

### ISSUE THREAD

**Issue by 0atman** [ryankemper/writings#2]
Could you summarise the article in the tldr section, as well as summarising the what COVID is?

**Comment by ryankemper** [comment]
Thanks @0atman 

By summarizing what COVID is, do you mean in the sense of, explaining that it's a respiratory disease and some of the symptoms?

As for summarizing the article - any ideas on what parts? Are you thinking more specifics about deleterious effects of lockdown, etc?

(Incidentally it might make sense for me to keep the current TL;DR as is and then have a more extended TL;DR, since the current iteration of the TLDR is meant to be simple enough for a school-aged child to understand hopefully)

**Comment by 0atman** [comment]
Yes, I think that the article could do with a tl;dr section that summarises not just COVID, but the various arguments you present about the problems we face during lockdown, and their counterpoints etc.

The current tl;dr is a good description of COVID. I think by this point, we're all pretty well-versed in the disease, so no need to expand that section. 

A strong addition would be to link to a good explainer video, such as the well-researched [Kuzdesagt video](https://www.youtube.com/watch?v=BtN-goy9VOY) on this topic, you could even embed it. You can tell that the video tries to use the same simple language strategy that you've used.

(By the way, there's a formal English subset called [Basic English](https://en.wikipedia.org/wiki/Basic_English) you could look into for this sort of thing, and perhaps the video uses it, or something like it.)

**Comment by 0atman** [comment]
> For those short on time or who have difficulty understanding technical language, you are advised to skip to “Section 6: TL;DR - Simple Words Only”, which explains the important parts in simple language that even children can understand.

Like, read this: Section 6 is just about COVID, not "the important parts".  *make it about the important parts!*

**Comment by ryankemper** [comment]
BTW just to circle back here, I totally agree that the section has a lot of potential that it's not remotely living up to currently. This is definitely a big area for improvement. I should be able to circle back within ~48 hours if someone else doesn't get to it.
